src/pages/Contact.js

```diff
@@ -0,0 +1,46 @@
+import React, { useState, useEffect } from "react";
+
+import '../index.css';
+import './About.css';
+import '../components/Footer.css'
+import {MatrixText} from "../components/MatrixText";
+
+import spotify from "../images/icons/Spotify.svg";
+import youtube from "../images/icons/youtube.svg";
+import tiktok from "../images/icons/Tik Tok.svg";
+import instagram from "../images/icons/Instagram.svg";
+import BasicPage from "../components/BasicPage";
+
+export const Contact = (props) => {
+
+  return (
+      <BasicPage>
+        <div>
+          <MatrixText text="for all speakerhug-related matters, contact: " length={45} speed={3}></MatrixText>
```

Comment:
ok this was my bad design but matrix text is "always" a link. so this should just be normal text imo. would look cooler if just hte email was obfuscated too, imo